api-tester/src/defaults/kotlin/com/revenuecat/apitester/kotlin/DeprecatedPurchasesAPI.kt

```diff
@@ -103,5 +103,8 @@ private class DeprecatedPurchasesAPI {
 
         Purchases.debugLogsEnabled = false
         val debugLogs: Boolean = Purchases.debugLogsEnabled
+
+        val finishTransactions: Boolean = purchases.finishTransactions
+        purchases.finishTransactions = true
```

Comment:
we should also add `syncAmazonObserverModePurchase` here

Reply:
Definitely! Added it to the Java API tests as well.